README.md

```diff
@@ -68,6 +68,11 @@ defmodule MyApp.Endpoint do
 end
 ```
 
+Wildcards can be used in the origins specification, by using regex:
+```elixir
+    origins: [~r{//foo.com}, ~r{//localhost}]
```

Comment:
Actually, maybe let's make the regex useful (because in the case above, `"http://foo.com"` would likely do the work as well :) Something like `~r{[a-zA-Z\-_]*\.example\.com}` to match all the subdomains of `example.com`. Wdyt?